src/content/guides/integrating-third-party-asset.mdx

```diff
@@ -108,6 +108,29 @@ curl https://auth.centrapay.com/api/.well-known/jwks.json
 }
 ```
 
+To verify a request start by decripting the JWT.
+
+**Decripted JWT Payload**
```

Comment:
@laurencemcknight There is a sentence above about verifying the JWT using the public key. I think this needs restructuring - worth looking at how other companies document this (e.g. [Stripe](https://stripe.com/docs/webhooks/signatures))